server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -1530,6 +1545,18 @@ private void setSearchThrottled(boolean searchThrottled) {
         this.searchThrottled = searchThrottled;
     }
 
+    /**
+     * Returns true if unreferenced files should be cleaned up on merge failure for this index.
+     *
+     */
+    public boolean shouldCleanupUnreferencedFile() {
```

Comment:
nit : `shouldCleanupUnreferencedFiles` everywhere.

Reply:
Addressed the comments